src/components/fields/BooleanField.js

```diff
@@ -56,6 +60,8 @@ if (process.env.NODE_ENV !== "production") {
     uiSchema: PropTypes.object,
     idSchema: PropTypes.object,
     onChange: PropTypes.func.isRequired,
+    onFocus: PropTypes.func.isRequired,
+    onBlur: PropTypes.func.isRequired,
```

Comment:
Are these really required?

Reply:
Agreed. Will back these out... I'd put them in there to keep the code similar to onChange event...